### PR TITLE
Fix UI tests

### PR DESCRIPTION
**Pull request by JavonDavis**
Fixes UI tests by removing old stats test code

To test: Ensure all the test pass

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by JavonDavis** [comment]
I was able to reproduce a failure and you can see why if you go to 48s on the video run here https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670/matrices/5838467063070142614/executions/bs.6c4729a635a03b55/videos, it's that previous issue I mentioned with the click, because of the flakiness of this click function, when elements behave different with a long click it can cause flakiness in the tests

**Comment by JavonDavis** [comment]
@jkmassel Could you take a look at these when you get a chance?

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="RIGHT_MARGIN" value="120" />
```
Comment (jkmassel) [inline .idea/codeStyles/Project.xml]: Can we undo this change? I don't think it's needed here :)

File: WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java [hunk 2]
```diff
@@ -73,38 +80,72 @@ public static boolean isElementCompletelyDisplayed(ViewInteraction element) {
...
+    }
+
+
+
+    private static boolean isResourceId(String text) {
+        return text.startsWith("id");
+    }
+
+    /**
+     * Uses UIAutomator to click on an element using either the resource ID or text in the cases of flakiness in
```
Comment (jkmassel) [inline WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java]: The bots aren't happy about this one – the line is too long – can we constrain it to 120 chars?

File: WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java [hunk 3]
```diff
@@ -383,6 +424,7 @@ public static FirstMatcher first() {
         return new FirstMatcher();
     }
 
+
```
Comment (jkmassel) [inline WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java]: Probably don't need the extra whitespace here?

File: WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java [hunk 4]
```diff
@@ -488,6 +530,7 @@ public boolean matchesSafely(View view) {
         };
     }
 
+
```
Comment (jkmassel) [inline WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java]: Probably don't need the extra whitespace here?

File: WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java (hunk 2 above)
Comment (JavonDavis) [reply WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java]: Weird 🤔 I checked and the length is 113 chars, I reduced it anyway to be on the safe side

File: WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java (hunk 3 above)
Comment (JavonDavis) [reply WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java]: Done

File: WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java (hunk 4 above)
Comment (JavonDavis) [reply WordPress/src/androidTest/java/org/wordpress/android/support/WPSupportUtils.java]: Done

File: .idea/codeStyles/Project.xml [hunk 5]
```diff
@@ -246,4 +246,4 @@
       <option name="ENUM_CONSTANTS_WRAP" value="5" />
     </codeStyleSettings>
   </code_scheme>
-</component>
\ No newline at end of file
+</component>
```
Comment (jkmassel) [inline .idea/codeStyles/Project.xml]: Can we remove this change too? 🙈

Comment (JavonDavis) [reply .idea/codeStyles/Project.xml]: I don't know why it keeps doing this, I was certain I updated my settings to avoid this but now this one is a weird one for the IDE to make.

Comment (jkmassel) [reply .idea/codeStyles/Project.xml]: Yeah it's super annoying. I'd suggest just throwing it away with your git client – I've had to do that a lot in the last few builds, too 🤔
